Cargo.toml

```diff
@@ -4,6 +4,7 @@ members = [
     "util",
     "hal-core",
     "hal-x86_64",
+    "hal-multiboot2",
```

Comment:
I'm guessing this got added to the commit accidentally?

Reply:
asdf